lib/client.js

```diff
@@ -55,6 +55,7 @@ class Client extends Emitter {
     return new Promise((resolve, reject) => {
       opts = opts || {} ;
       const cookie = uuid() ;
+      name = name.replace(/([A-Z])/g, ' $1').toLowerCase();
```

Comment:
This line transforms camel case to regular string. This is a byproduct of the fact that function names cant have spaces in them